### PR TITLE
Update indicator tests

### PR DESCRIPTION
**Pull request by TiKeil**
This pull request adds a test for the global indicator function. This test first showed that float compare in dune-xt-common has a bug. This resulted in a small peak in the lower left corner.

This bug is now fixed as you can compare the visualized functions for indicator function and indicator grid function.

### CODE REVIEW

File: dune/xt/functions/test/indicator_function.tpl [hunk 1]
```diff
@@ -126,20 +128,18 @@ TEST_F(IndicatorFunction_from_{{GRIDNAME}}_to_{{r}}_times_{{rC}}, global_evaluat
     DomainType lower_left(ll);
     for (auto ur : {1., 0.75, 0.5, 0.25}) {
```
Comment (renefritze) [inline dune/xt/functions/test/indicator_function.tpl:129]: Just FYI, you can use our ```XT::Common::valueRange(start,stop,delta)``` in these kinds of situations.

Comment (TiKeil) [reply dune/xt/functions/test/indicator_function.tpl:129]: Great thanks

**Comment by TiKeil** [comment]
I can merge this, right?

**Comment by renefritze** [comment]
I favor "rebase and merge", but yes.

**Pull request merged**